src/dynapi/SDL_dynapi.sym

```diff
@@ -0,0 +1,871 @@
+    SDL_IsGameController;
+    SDL_GameControllerNameForIndex;
+    SDL_GameControllerPathForIndex;
+    SDL_GameControllerTypeForIndex;
+    SDL_GameControllerMappingForDeviceIndex;
+    SDL_GameControllerOpen;
+    SDL_GameControllerFromInstanceID;
+    SDL_GameControllerFromPlayerIndex;
+    SDL_GameControllerName;
+    SDL_GameControllerPath;
+    SDL_GameControllerGetType;
+    SDL_GameControllerGetPlayerIndex;
+    SDL_GameControllerSetPlayerIndex;
+    SDL_GameControllerGetVendor;
+    SDL_GameControllerGetProduct;
+    SDL_GameControllerGetProductVersion;
+    SDL_GameControllerGetFirmwareVersion;
+    SDL_GameControllerGetSerial;
+    SDL_GameControllerGetAttached;
+    SDL_GameControllerGetJoystick;
+    SDL_GameControllerEventState;
+    SDL_GameControllerUpdate;
+    SDL_GameControllerGetAxisFromString;
+    SDL_GameControllerGetStringForAxis;
+    SDL_GameControllerGetBindForAxis;
+    SDL_GameControllerHasAxis;
+    SDL_GameControllerGetAxis;
+    SDL_GameControllerGetButtonFromString;
+    SDL_GameControllerGetStringForButton;
+    SDL_GameControllerGetBindForButton;
+    SDL_GameControllerHasButton;
+    SDL_GameControllerGetButton;
+    SDL_GameControllerGetNumTouchpads;
+    SDL_GameControllerGetNumTouchpadFingers;
+    SDL_GameControllerGetTouchpadFinger;
+    SDL_GameControllerHasSensor;
+    SDL_GameControllerSetSensorEnabled;
+    SDL_GameControllerIsSensorEnabled;
+    SDL_GameControllerGetSensorDataRate;
+    SDL_GameControllerGetSensorData;
+    SDL_GameControllerGetSensorDataWithTimestamp;
+    SDL_GameControllerRumble;
+    SDL_GameControllerRumbleTriggers;
+    SDL_GameControllerHasLED;
+    SDL_GameControllerHasRumble;
+    SDL_GameControllerHasRumbleTriggers;
+    SDL_GameControllerSetLED;
+    SDL_GameControllerSendEffect;
+    SDL_GameControllerClose;
+    SDL_GameControllerGetAppleSFSymbolsNameForButton;
+    SDL_GameControllerGetAppleSFSymbolsNameForAxis;
+    SDL_ReportAssertion;
+    SDL_SetAssertionHandler;
+    SDL_GetDefaultAssertionHandler;
+    SDL_GetAssertionHandler;
+    SDL_GetAssertionReport;
+    SDL_ResetAssertionReport;
+    SDL_NumHaptics;
+    SDL_HapticName;
+    SDL_HapticOpen;
+    SDL_HapticOpened;
+    SDL_HapticIndex;
+    SDL_MouseIsHaptic;
+    SDL_HapticOpenFromMouse;
+    SDL_JoystickIsHaptic;
+    SDL_HapticOpenFromJoystick;
+    SDL_HapticClose;
+    SDL_HapticNumEffects;
+    SDL_HapticNumEffectsPlaying;
+    SDL_HapticQuery;
+    SDL_HapticNumAxes;
+    SDL_HapticEffectSupported;
+    SDL_HapticNewEffect;
+    SDL_HapticUpdateEffect;
+    SDL_HapticRunEffect;
+    SDL_HapticStopEffect;
+    SDL_HapticDestroyEffect;
+    SDL_HapticGetEffectStatus;
+    SDL_HapticSetGain;
+    SDL_HapticSetAutocenter;
+    SDL_HapticPause;
+    SDL_HapticUnpause;
+    SDL_HapticStopAll;
+    SDL_HapticRumbleSupported;
+    SDL_HapticRumbleInit;
+    SDL_HapticRumblePlay;
+    SDL_HapticRumbleStop;
+    SDL_GetVersion;
+    SDL_GetRevision;
+    SDL_GetPowerInfo;
+    # extra symbols go here
```

Comment:
The perl script will do a regex replace of `#extra symbols go here` with the new symbol(s).
But a perl export should probably rewrite it to make it perly.